scripts/build_layers.sh

```diff
@@ -40,9 +40,9 @@ function docker_build_zip {
     docker build -t datadog-lambda-layer-node:$1 . --no-cache \
         --build-arg image=node:$1-alpine
 
-    # Run the image by runtime tag, tar its generatd `node` directory to sdout,
-    # then extract it to a temp directory.
-    docker run --rm datadog-lambda-layer-node:$1 tar cf - /nodejs | tar -xf - -C $temp_dir
+    # Run the image by runtime tag and copy the output /nodejs to the temp dir
```

Comment:
For more information, check the python PR: https://github.com/DataDog/datadog-lambda-python/commit/8f2686cdcbc8d25b122e3ba24bdec3e5dd3a8f7d